clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/RocksDBMemoryStats.java

```diff
@@ -89,6 +90,9 @@ public RocksDBMemoryStats(MetricsRepository metricsRepository, String name, bool
         // Lock down the list of RocksDB interfaces while the collection is ongoing
         synchronized (hostedRocksDBPartitions) {
           for (RocksDBStoragePartition dbPartition: hostedRocksDBPartitions.values()) {
+            if (dbPartition instanceof ReplicationMetadataRocksDBStoragePartition) {
```

Comment:
Why skipping?
IIUC, the instance of `ReplicationMetadataRocksDBStoragePartition` contains two CFs: default and RMD and here we should continue to call `dbPartition.getRocksDBStatValue(metric);` to measure the default CF usage, right?

That means we should remove this `if` statement.